src/__tests__/appbar.js

```diff
@@ -0,0 +1,11 @@
+import Appbar from '../components/Appbar';
+import renderWithProviders from '../utils/test-utils';
+
```

Comment:
- [ ]  The application should have: 
-  unit tests for several components of the app and some pure functions in Redux code.
-  integration tests using React Testing Library

Kindly add a couple of unit tests.